aiida/manage/tests/unittest_classes.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""
+Test classes and test runners for testing AiiDA plugins with unittest.
+"""
+import unittest
+import warnings
+
+from aiida.common.warnings import AiidaDeprecationWarning
+from aiida.manage.manager import get_manager
+from . import _GLOBAL_TEST_MANAGER, test_manager, get_test_backend_name, get_test_profile_name
+
+__all__ = ('PluginTestCase', 'TestRunner')
+
+warnings.warn(  # pylint: disable=no-member
+    'This module has been deprecated and will be removed soon. Please use the `pytest` fixtures instead.',
```

Comment:
We should definitely mention fixtures but there might be even a path that involves less code changes using things like https://docs.pytest.org/en/6.2.x/xunit_setup.html

I will play around with this in the plugin-cutter and come back to this PR.

Reply:
Is there already a page somewhere with instructions on how to migrate? Then we could add that in the deprecation warning.